### ISSUE TITLE
train dataset

### ISSUE THREAD

**Issue by cbxgss** [GAIR-NLP/DeepResearcher#19]
Great work! I really appreciate the effort you've put into cleaning up the QA datasets. Given that many of the original datasets contain questions of varying quality, the dataset you've provided is especially valuable.

However, I noticed the following statement in your paper:

> After applying these quality control measures, we constructed a final training dataset of 80,000 examples with a distribution ratio of 1:1:3:3 for NQ:TQ:HotpotQA:2Wiki.

When I examined the actual training set (`data/train.parquet`), I found the distribution to be:

- NQ: 37,946
- TQ: 17,131
- 2Wiki: 24,923

This raised a question for me: Is there an updated version of the training dataset with a higher proportion of multi-hop questions (like HotpotQA and 2Wiki)?
Or did I possibly miss an additional data file?

Thanks again for your great work, and I'm looking forward to your response!

**Comment by Fu-Dayuan** [comment]
Thanks! The last training data do have some mistakes in the data source's label. We've corrected it!

Now it has:

```
{'hotpotqa': 35077, '2wiki': 24923, 'nq': 10000, 'tq': 10000}
```

There is a little bit more hotpotqa and less 2wiki just because 2wiki has more "yes or no" questions, so most of its data was filtered.

**Comment by cjfcsjt** [comment]
May I ask if the train set is uploaded? I clone the project and still find the same issue
